src/show/components/Navbar/index.js

```diff
@@ -1,63 +1,175 @@
+                    placeholder='Username'
+                  />
+                </div>
+                <div>
+                  <label htmlFor='email-address' className='sr-only'>
+                    Email address
+                  </label>
+                  <input
+                    id='email-address'
+                    name='email'
+                    type='email'
+                    autoComplete='email'
+                    required
+                    className='appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 focus:outline-none focus:ring-primary-500 focus:border-primary-500 focus:z-10 text-sm'
+                    placeholder='Email address'
+                  />
+                </div>
 
-        <div className='flex border-2 border-primary-50 items-center hover:border-2 hover:rounded-full hover:border-white transition-all duration-300 hover:px-2 hover:py-1'>
-          <Button
-            variant='secondary'
-            iconLeft={<TbStairsUp className='px-0 mx-0' size='1.5rem' />}
-            label='raise it'
-          />
+                <div>
+                  <label htmlFor='password' className='sr-only'>
+                    Password
+                  </label>
+                  <input
+                    id='password'
+                    name='password'
+                    type='password'
+                    autoComplete='current-password'
+                    required
+                    className='appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900  focus:outline-none focus:ring-primary-500 focus:border-primary-500 focus:z-10 text-sm'
+                    placeholder='Password'
+                  />
+                </div>
+
+                <div>
+                  <label htmlFor='confirmpassword' className='sr-only'>
+                    Confirm Password
+                  </label>
+                  <input
+                    id='confirmpassword'
+                    name='confirmpassword'
+                    type='password'
+                    autoComplete='current-password'
+                    required
+                    className='appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-b-md focus:outline-none focus:ring-primary-500 focus:border-primary-500 text-sm'
+                    placeholder='Confirm password'
+                  />
+                </div>
+              </div>
+
+              <div className='flex items-center justify-between'></div>
+              <div>
+                <button
+                  type='submit'
+                  className='group relative w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-primary-500 hover:bg-primary-600'>
+                  Sign Up
+                </button>
```

Comment:
You have a button component, same bellow.

Reply:
you are right